src/DynamoManipulation/DynamoManipulationExtension.cs

```diff
@@ -128,14 +128,14 @@ public void Startup(ViewStartupParams viewStartupParams)
             manipulatorDaemon = ManipulatorDaemon.Create(new NodeManipulatorFactoryLoader());
         }
 
-        public void Loaded(ViewLoadedParams viewStartupParams)
+        public void Loaded(ViewLoadedParams loadedParams)
```

Comment:
I guess this won't get rid of the code smell either. Reverting ...

Reply:
This will not.. Need to be consistent with the interface definition I believe